extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -379,9 +377,43 @@ describes.realWin(
       });
 
       expect(openStub).calledWithExactly(
-        'https://example.com',
+        'https://advertiser.example',
         '_blank',
-        'noopener,attributiondestination=https://example.com,attributionsourceeventid=EFnZ8GunL1xrwNTIHbXrvQ==,attributionreportto=https://google.com'
+        'noopener,attributionsrc=https%3A%2F%2Fadtech.example'
+      );
+    });
+
+    it('should handle empty attributionsrc when given `browserAdConversion`', async () => {
+      env.sandbox
+        .stub(AmpAdExit.prototype, 'detectAttributionReportingSupport')
+        .returns(true);
+      const openStub = env.sandbox.stub(win, 'open').returns(win);
+      const config = {
+        targets: {
+          landingPage: {
+            finalUrl: 'https://advertiser.example',
```

Comment:
When attributionsrc='', then finalUrl should be the full url starting with adtech domain.

Reply:
Done.